src/protocols/http/hpr_roaming_utils.erl

```diff
@@ -0,0 +1,80 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 17. Sep 2022 10:56 AM
+%%%-------------------------------------------------------------------
+-module(hpr_roaming_utils).
+-author("jonathanruttenberg").
+
+%% API
+-export([
+    hexstring/1,
+    binary_to_hexstring/1,
+    format_time/1,
+    uint32/1,
+    hexstring_to_binary/1,
+    hexstring_to_int/1,
+    hexstring/2,
+    hex_to_binary/1
+]).
+
+-spec binary_to_hex(binary()) -> binary().
+binary_to_hex(ID) ->
+    <<<<Y>> || <<X:4>> <= ID, Y <- integer_to_list(X, 16)>>.
```

Comment:
Same as `binary:encode_hex/1`?

Reply:
It is. Now that we're using OTP24, we can use `binary:encode_hex/1`.